.clang-format-compare.py

```diff
@@ -0,0 +1,56 @@
+## @file      .clang-format-compare.py
+#  @brief     Tool for comparing source files with a formatted variant in order to detect clang-formatting
+#  @author    Julian van Doorn
+#  @license   See LICENSE
+
```

Comment:
Big changes to build environment in a different branch **next time**, then pull that branch towards the dev and dev towards your WIP feature so the updates don't cloud the commits.